Mathlib/Probability/Moments.lean

```diff
@@ -184,6 +185,9 @@ theorem mgf_neg : mgf (-X) μ t = mgf X μ (-t) := by simp_rw [mgf, Pi.neg_apply
 
 theorem cgf_neg : cgf (-X) μ t = cgf X μ (-t) := by simp_rw [cgf, mgf_neg]
 
+theorem mgf_smul_left (α : ℝ) : mgf (α • X) μ t = mgf X μ (α * t) := by
+  simp_rw [mgf, Pi.smul_apply, smul_eq_mul, mul_comm α t, mul_assoc]
+
```

Comment:
I am confused why this is flagged as a new change given https://github.com/leanprover-community/mathlib4/pull/19798...

Reply:
If you merge master into the branch of this PR it should go away.